conbench/tests/api/test_results.py

```diff
@@ -321,167 +319,104 @@ class TestBenchmarkList(_asserts.ListEnforcer):
     url = "/api/benchmarks/"
     public = True
 
-    def test_benchmark_list(self, client):
-        self.authenticate(client)
-        benchmark_result = _fixtures.benchmark_result()
-        response = client.get("/api/benchmarks/")
-        self.assert_200_ok(response, contains=_expected_entity(benchmark_result))
-
-    def test_benchmark_list_filter_by_name(self, client):
-        self.authenticate(client)
-        _fixtures.benchmark_result(name="aaa")
-        benchmark_result = _fixtures.benchmark_result(name="bbb")
-        _fixtures.benchmark_result(name="ccc")
-        response = client.get("/api/benchmarks/?name=bbb")
-        self.assert_200_ok(response, [_expected_entity(benchmark_result)])
-
-    def test_benchmark_list_filter_by_batch_id(self, client):
-        self.authenticate(client)
-        benchmark_result = _fixtures.benchmark_result(batch_id="20")
-        response = client.get("/api/benchmarks/?batch_id=20")
-        self.assert_200_ok(response, [_expected_entity(benchmark_result)])
+    def _make_request(self, client, **kwargs) -> dict:
+        query_args = [f"{key}={value}" for key, value in kwargs.items() if value]
```

Comment:
1. The `if value` makes me nervous if we try to use this to override a non-null value with null
    1. If this gets removed, we need to jsonify `None` into `null`
2. Do strings need quoting here?

If nothing is breaking, maybe just a comment for future debugging

Reply:
Maybe I'll change this to `if value is not None` and clarify with a docstring. Since this is a test, I knew exactly all the possible kwargs that would be provided to this method, so it's fine now, but you're right we should make sure it's easy to debug if someone changes the test.